[Do Not Merge] Announce NRQL Results Limit Increases (#15072)
newrelic/docs-website: src/content/whats-new/2023/11/whats-new-11-07-nrql-limit-increases.md

---
title: 'NRQL Query Result Limits are Increasing'
summary: 'The maximum result limit for NRQL queries is increasing from 2000 to 5000'
releaseDate: '2023-11-07'
learnMoreLink: 'https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-limit'
---

> NRQL Query Result Limits are increasing! When querying New Relic with NRQL, the `LIMIT` clause is used to specify the maximum number of results to be returned from a query. The maximum allowable value has been 2000, and this is increasing to a value of 5000. This increase will allow you to get more from your NRQL Queries in your dashboards, Query Builder, and more.

Reviewer: ```suggestion
The previous maximum of 2000 results is now increasing to a new maximum of 5000. 
```
